ground_station/src/utils.cpp

```diff
@@ -202,6 +186,28 @@ int32_t Utils::getFlashMemoryUsage() {
   return static_cast<int32_t>(std::ceil(percentage));
 }
 
+void Utils::streamUsb(Telemetry *link, uint8_t link_idx) {
+  char print_char[150];
+
+  const int32_t time_int = link->data.ts() / 10;
+  const int32_t time_prec = link->data.ts() - time_int * 10;
+
+  const auto lat_int = static_cast<int32_t>(link->data.lat());
+  const auto lat_prec = static_cast<int32_t>((link->data.lat() - static_cast<float>(lat_int)) * 100000.0F);
+
+  const auto lon_int = static_cast<int32_t>(link->data.lon());
+  const auto lon_prec = static_cast<int32_t>((link->data.lon() - static_cast<float>(lon_int)) * 100000.0F);
+
+  const auto voltage_int = static_cast<int32_t>(link->data.voltage());
+  const auto voltage_prec = static_cast<int32_t>((link->data.voltage() - static_cast<float>(voltage_int)) * 10.0F);
+
+  std::snprintf(print_char, 150,
+                "Link %d: Ts: %ld.%ld, State: %d, Lat: %ld.%05ld, Lon: %ld.%05ld, Alt: %ld, Vel: %d, V: %ld.%ld",
```

Comment:
can you paste some sample output

Reply:
![image](https://github.com/catsystems/cats-embedded/assets/36957747/45f16d4c-659c-4b02-8701-d75c9b5468d7)